src/main/java/com/uid2/shared/vertx/VertxUtils.java

```diff
@@ -78,6 +79,21 @@ public static ConfigRetriever createConfigRetriever(Vertx vertx) {
         return ConfigRetriever.create(vertx, retrieverOptions);
     }
 
+    public static AbstractMap.SimpleEntry<String, String> parseClientAppVersion(String appVersions) {
```

Comment:
Could this instead be `Map.Entry<String, String>`?

Reply:
Done